packages/gatsby/src/utils/gatsby-dependents.js

```diff
@@ -0,0 +1,13 @@
+import { store } from "../redux"
+import rpt from "read-package-tree"
+
+// Returns [Object] with name and path
+module.exports = async () => {
+  const { program } = store.getState()
+  const allNodeModules = await rpt(program.directory)
+  return allNodeModules.children.filter(
+    node =>
+      (node.package.dependencies && node.package.dependencies[`gatsby`]) ||
```

Comment:
Doing this and not name based because this let’s us check version for v3 in the future 

Cc @KyleAMathews @pieh